### ISSUE TITLE
App Clear is Extremely slow after i updated to the latest version.

### ISSUE THREAD

**Issue by BlazeJatin** [d4rken/sdmaid-public#5344]
**Describe the bug**
I use the latest sd maid from Play Store i vene have the pro version installed. SD maid is extremely slow in clearing app cache in app cleaner and storage also loads very slow and takes time sorting it.

**Expected behavior**
I earlier versions i could see the app automating and clearing cache one by one and fairly quick. Now i just see a plain opaque screen with the maid animation.

**Screenshots**

**Debug log**
https://file.io/i6fGdbrxeN6V

**Smartphone (please complete the following information):**
 - Device: Samsung A20s
 - OS: Android 11 stock one ui

**Comment by BlazeJatin** [comment]
Its just not cleaning anything.

**Comment by BlazeJatin** [comment]
https://www.filemail.com/d/wxsikyyndfpdxsq here is the log file

**Comment by d4rken** [comment]
> after i updated to the latest version 

Latest version of what? SD Maid?
Android?

**Comment by d4rken** [comment]
What was the last version that worked, as far as I remember nothing changed to affect this in the last weeks.

Based on the log SD Maid is either not recognizing that the system window is the right one, or not getting the data from the system.

Can you record a short video of what the log shows?

**Comment by BlazeJatin** [comment]
https://fromsmash.com/Yfkh7kPs~p-bt

**Comment by d4rken** [comment]
The accessibility sequence should be empty, if you changed that, then it's the cause for being slow. It's just a workaround for unsupported ROMs, you can enter text and SD Maid will click it.

Your video looks like something else though, the screen stays gray as if the system settings could not be launched?

**Comment by BlazeJatin** [comment]
The accessibility sequence is empty.
I just tried going through some app infos to clear cache manually and it still loads extremely slow and crashes.
Maybe the problem was with my device. It never happened though.
I  reseted app preferences and that day it all started happening idk how to fix it now I've tried clearing cache partition, resetting all settings. Maybe I'll have to factory reset my device.

I'm gonna close this issue because the problem is from my side.
